plugin.video.cbc-sports/addon.py

```diff
@@ -0,0 +1,495 @@
+import xbmcaddon, urllib.request, urllib.parse, urllib.error, xbmcgui, xbmcplugin, re, sys, os
+from bs4 import BeautifulSoup
+import html5lib
+import json
+from datetime import datetime, timedelta as td
+import time
+
+_addon = xbmcaddon.Addon()
+_addon_path = _addon.getAddonInfo('path')
+selfAddon = xbmcaddon.Addon(id='plugin.video.cbc-sports')
+sys.path.append(os.path.join( _addon_path, 'resources', 'lib' ) )
```

Comment:
please avoid using sys.path.append, use relative imports instead:

`from resouces.lib import x`

Reply:
This has been changed to a relative path and sys.path.append has been removed.